sql/wsrep_trans_observer.h

```diff
@@ -433,6 +433,16 @@ static inline void wsrep_close(THD* thd)
   DBUG_VOID_RETURN;
 }
 
+static inline void wsrep_cleanup(THD* thd)
+{
```

Comment:
would it be sufficient to do this client state cleanup as part of wsrep_close() ?
If not, I suggest changing the name of this function to narrow down to client state cleanup only, e.g.: wsrep_client_state_cleanup

Reply:
It's best to leave wsrep_cleanup() as is. There reason is that we do have wsrep_cleanup() in mysql-wsrep 8.0. I think it is better to leave them the same.